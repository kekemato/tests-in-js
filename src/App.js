import React from 'react'

const App = props => (
    <div>
        <h1>Hello Jest!</h1>
    </div>
)

export default App