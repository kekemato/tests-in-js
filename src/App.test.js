import rerenderer from 'react-test-renderer'
import React from 'react'

import App from './App'

test('renders correctly', () => {
    const tree = rerenderer
    .create(<App />)
    .toJSON()

    expect(tree).toMatchSnapshot()
})