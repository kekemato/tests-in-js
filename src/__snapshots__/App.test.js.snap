// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Hello Jest!!!
  </h1>
</div>
`;
